services/clusters-keeper/src/simcore_service_clusters_keeper/rpc/clusters.py

```diff
@@ -37,17 +39,16 @@ async def get_or_create_cluster(
         ec2_instance = new_ec2_instances[0]
     assert ec2_instance is not None  # nosec
     app_settings = get_application_settings(app)
-    cluster_get = ClusterGet.from_ec2_instance_data(
+    return create_cluster_from_ec2_instance(
         ec2_instance,
         user_id,
         wallet_id,
         app_settings.CLUSTERS_KEEPER_COMPUTATIONAL_BACKEND_GATEWAY_PASSWORD,
+        gateway_ready=bool(
```

Comment:
Q: so if `gateway_ready==False` how does the system reponds? i.e. what is the workflow in that situation? do you retry?

Reply:
it retries again 5 seconds later